### PR TITLE
Fixed accordion not opening properly

### PR DESCRIPTION
**Pull request by mbuehrig**

### CODE REVIEW

File: src/components/Accordion/accordion.scss [hunk 1]
```diff
@@ -41,10 +43,9 @@
 
   &__panel {
     overflow: hidden;
+    transition: max-height $accordion-animation-duration $accordion-animation-easing;
 
-    &[ aria-hidden='true' ] {
-      max-height: 0;
-    }
+    max-height: 0;
```
Comment (tschuems) [inline src/components/Accordion/accordion.scss:48]: this is the only change you have to do to get the accordion to work. 
the problem was, that the `accordion__panel` had a `max-height` of `auto` as standard. it needs the `max-height` of `0`.

Comment (tschuems) [reply src/components/Accordion/accordion.scss:48]: so we can say, you can revert everything else 👍 😄

Comment (mbuehrig) [reply src/components/Accordion/accordion.scss:48]: No, still better solution now, without anime.js and therefore slimmer.

Comment (tschuems) [reply src/components/Accordion/accordion.scss:48]: true that, but we decided once to include animejs to show a example in javascript. but anyways i will accept this.

**Review by tschuems — APPROVED** [review]

**Pull request merged**